api-reference/10 UI Components/GridBase/1 Configuration/onRowInserting.md

```diff
@@ -28,64 +28,19 @@ The data of the row that should be inserted.
 Model data. Available only if you use Knockout.
 
 ---
-[note] In batch [editing mode](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/editing/mode.md '{basewidgetpath}/Configuration/editing/#mode'), this function is executed for each row individually if several rows should be inserted.
 
-The following code shows how to use the function parameter's **cancel** field to prevent or continue row insertion. In this code, a Promise is assigned to this field. Row insertion continues if row data validation on the server succeeds (the Promise is resolved); otherwise, row insertion is prevented (the Promise is rejected).
+This function allows you to intercept row insert and perform additional actions. The following code shows how to use the function parameter's **cancel** field to prevent or continue row insert. In this code, a Promise is assigned to this field. Row insert continues if a user confirms it and row data validation on the server succeeds (the Promise is resolved); otherwise, row insert is prevented (the Promise is rejected).
```

Comment:
I think your original version "row insertion" is better than "row insert", since "insert" is a verb, but "insertion" is a noun that means the act of inserting something. Can we change this to the following?

`This function allows you to intercept row insertion and perform additional actions. The following code shows how to use the function parameter's **cancel** field to prevent or continue row insertion. In this code, a Promise is assigned to this field. Row insertion continues if a user confirms it and row data validation on the server succeeds (the Promise is resolved); otherwise, row insertion is prevented (the Promise is rejected).`